deployment/helm/posix-mapper/templates/posix-mapper-tomcat-expose.yaml

```diff
@@ -10,6 +10,10 @@ spec:
   - port: 8080
     name: http-connection
     protocol: TCP
+  - port: 80
+    targetPort: 8080
+    name: http-connection-automatic
+    protocol: TCP
```

Comment:
I guess this won't hurt, but we do install Traefik in front of these services which should proxy to the 8080 port.  Is this just to enable Port 80 without a proxy service?